### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by aloysb** [zk-org/zk#319] [Share]
Hey @mickael-menu ,

This is looking absolutely amazing, it is what I hope Obsidian would become until they have to close their source code (which I understand).

Thank you so much for your work.

**Comment by mickael-menu** [comment]
I appreciate the kind words, happy to hear you enjoy using `zk`.

**Comment by zalegrala** [comment]
I also love this project.  I use it every day.  Thanks for the hard work.
